pages/integrations/sso.md.erb

```diff
@@ -89,3 +89,18 @@ No, SSO providers are setup using a unique identifier and are unaffected when a
 
 ### Can I merge two organizations that use different SSO providers?  
 In short, yes, you can. However, merging Buildkite organizations that already have SSO providers might be a tricky scenario, and it's highly recommended that you contact support@buildkite.com for help or guidance before you attempt such migration.  
+
+### Why am I being asked for my password in the "Authorization Required" screen on top of signing in to Buildkite using my organization's SSO? Can this authorization process be simplified?  
+The short answer is no. Signing in to your Buildkite organization requires both authorization and authentication procedures via Buildkite and your SSO provider. The two-factor authentication confirms that you are who you claim to be. Authorization (your password) verifies that you have permission to access and perform various actions in this Buildkite organization.  
```

Comment:
I think with a short answer / long answer type approach we need to make sure we're still being friendly with it.

Reply:
Friendlified and (hopefully) placated the linter.